posts/2021-08-31-liberty-bikes.adoc

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Liberty Bikes: A Microservices-Based Game Built On Open Liberty"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/realModusOperandi
+author_github: https://github.com/realModusOperandi
+seo-title: Liberty Bikes: A Microservices-Based Game Built On Open Liberty - OpenLiberty.io
+seo-description: See how a multiplayer browser game can be implemented as microservices with a Open Liberty and MicroProfile.
+blog_description: See how a multiplayer browser game can be implemented as microservices with a Open Liberty and MicroProfile.
+open-graph-image: https://openliberty.io/img/blog/lb-title.png
+---
+= Liberty Bikes: A Microservices-Based Game Built On Open Liberty
+Liam Westby <https://github.com/realModusOperandi>
+:imagesdir: /
+:url-prefix: https://openliberty.io
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+[#Intro]
+
+image::img/blog/lb-title.png[Liberty Bikes, built on Open Liberty, align="center"]
+
+A while ago, the Open Liberty team was looking for some cool and interesting demos: projects built to take advantage of and show off some of the new features in Open Liberty, including MicroProfile. Demos that could help illustrate key concepts provided by this great technology and could be used as tutorials to show developers how to do cool things in their own code.  That sounded like a fun project! A team of us (Olen Cook, Ryan Esch, Andy Guibert, and I, with art by Sarah Ishida) got together to come up with a submission. 
+
+But what to create? It should be more complex and sophisticated than Hello World, as well as being polished and interesting enough to draw people in who might be passing by a conference booth. Something that keeps bringing people back, not just to have a look at the code, but to keep using it again and again. 
+
+Something like... a game! We call it Liberty Bikes, and you can play it right here: link:http://libertybikes.mybluemix.net[Liberty Bikes]. It's all open source, so you can see how it works here: link:https://github.com/OpenLiberty/liberty-bikes[Liberty Bikes on GitHub]
+
+[#About]
+== What Is Liberty Bikes?
+
+image::img/blog/lb-game-screen.png[screen capture of the Liberty Bikes game board, align="center"]
+
+link:http://libertybikes.mybluemix.net[Liberty Bikes] is a multiplayer game for up to four players (either on four separate computers or with four mobile device controllers and one hosting computer) in which you pilot a bike that leaves a trail as it moves around the arena. Your objective is to be the last player standing, by avoiding other players' trails, the walls, and the obstacles scattered throughout the arena. You can steer your bike left or right, but you can't slow down or stop. 
```

Comment:
don't need a comma after "by"

Your objective is to be the last player standing by avoiding other...